src/main/haskell/language-kore/app/parser/Main.hs

```diff
@@ -1,123 +1,129 @@
 -}
 
--- | Main options record    
+-- | Main options record
 data KoreParserOptions = KoreParserOptions
-    { fileNameArgument  :: String -- ^ Filename to parse and verify 
-    , willPrintOption   :: Bool   -- ^ Option to print definition
-    , willVerifyOption  :: Bool   -- ^ Option to verify definition
-    , willChkAttrOption :: Bool   -- ^ Option to check attributes during verification
+    { fileName    :: !String -- ^ Filename to parse and verify
+    , willPrint   :: !Bool   -- ^ Option to print definition
+    , willVerify  :: !Bool   -- ^ Option to verify definition
+    , willChkAttr :: !Bool   -- ^ Option to check attributes during verification
     }
 
-                    
-{-| Command Line Argument Parser -}
+-- | Command Line Argument Parser
 commandLineParser :: Parser KoreParserOptions
-commandLineParser = KoreParserOptions
-      <$> argument str
-              ( metavar "FILE"
-              <> help "Kore source file to parse [and verify]" )
-      <*> switch3 "print"
-              "Print parsed definition to stdout" 
-              "Do not print parsed definition to stdout" 
-      <*> switch3 "verify"
-              "Verify well-formedness of parsed definition"
-              "Do not verify well-formedness of parsed definition" 
-      <*> switch3 "chkattr"
-              "Check attributes during verification"
-              "Ignore attributes during verification"
-
+commandLineParser =
+    KoreParserOptions
+    <$> argument str
+        (  metavar "FILE"
+        <> help "Kore source file to parse [and verify]" )
+    <*> switch3 "print"
+        "Print parsed definition to stdout [default]"
+        "Do not print parsed definition to stdout"
+    <*> switch3 "verify"
+            "Verify well-formedness of parsed definition [default]"
+            "Do not verify well-formedness of parsed definition"
+    <*> switch3 "chkattr"
+            "Check attributes during verification [default]"
+            "Ignore attributes during verification"
 
 -- | Run argument parser for kore-parser
 commandLineParse :: IO KoreParserOptions
 commandLineParse = execParser opts
-  where
-    opts = info (commandLineParser <**> helper)
-      ( fullDesc
-     <> progDesc "Parses Kore definition in FILE; optionally, \
-                 \Verifies well-formedness"
-     <> header "kore-parser - a parser for Kore definitions" )
+    where
+      opts = info
+             ( commandLineParser <**> helper )
+             (  fullDesc
+             <> progDesc "Parses Kore definition in FILE; optionally, \
+                         \Verifies well-formedness"
+             <> header "kore-parser - a parser for Kore definitions" )
 
 
 main :: IO ()
 main =
     do {
     ; KoreParserOptions
-          { fileNameArgument  = fileName
-          , willPrintOption   = willPrint 
-          , willVerifyOption  = willVerify
-          , willChkAttrOption = willChkAttr
-          } <- commandLineParse
+      { fileName    = fileName
```

Comment:
@bmmoore would suggest to also use record puns https://downloads.haskell.org/~ghc/7.2.1/docs/html/users_guide/syntax-extns.html#record-puns 
in combination with DuplicateRecord fields, especially for a use-case like yours below.